### PR TITLE
PIM-5434: When no access to any category, I can't edit my profile

### PR DESCRIPTION
**Pull request by aRn0D**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | yes
| Added Behats                      | no
| Changelog updated                 | no
| Review and 2 GTM                  | yes
| Micro Demo to the PO (Story only) | no 
| Migration script                  | no
| Tech Doc                          | no

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Doctrine/ORM/Repository/AttributeRepository.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ public function __construct(UserContext $userContext, EntityManager $em, $class)
     public function findTranslatedLabels(array $options = [])
     {
         $queryBuilder = $this->createQueryBuilder('a')
```
Comment (aRn0D) [inline src/Pim/Bundle/EnrichBundle/Doctrine/ORM/Repository/AttributeRepository.php]: Related to PIM-5536 too.

**Comment by damien-carcel** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Form/Type/ProductGridFilterChoiceType.php [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+     */
+    protected function getAttributesAsFilter()
+    {
+        return $this->attributeProvider->findTranslatedLabels(['useable_as_grid_filter' => true]);
+    }
+
+    /**
+     * @return array
+     */
+    protected function getSystemFilters()
```
Comment (skeleton) [inline src/Pim/Bundle/EnrichBundle/Form/Type/ProductGridFilterChoiceType.php]: can you add a description to explain it please ("System" is not very clear) ?

File: src/Pim/Bundle/EnrichBundle/Form/Type/ProductGridFilterChoiceType.php [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+use Oro\Bundle\DataGridBundle\Datagrid\Manager;
+use Pim\Component\Enrich\Provider\TranslatedLabelsProviderInterface;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Choice Type which displays product grid filter (system and attribute) grouped by attribute group
+ *
+ * @author    Arnaud Langlade <arnaud.langlade@akeneo.com>
+ * @copyright 2015 Akeneo SAS (http://www.akeneo.com)
```
Comment (skeleton) [inline src/Pim/Bundle/EnrichBundle/Form/Type/ProductGridFilterChoiceType.php]: 2016

**Comment by skeleton** [comment]
GTM

**Pull request merged**